misc/tools/atrium-samples-test/build.gradle

```diff
@@ -0,0 +1,7 @@
+task run {
+    dependsOn gradle.includedBuild('junit5').task(':test')
```

Comment:
I am afraid but I made a wrong assumption. I did not have a look at how composite builds work in gradle but they work the opposite way. I.e. the project which depends on another needs to do the includeBuild. Which means, we don't need a `atrium-samples-test` at all because not atrium has a dependency but the opposite. Thus remove the whole `atrium-samples-test` again. Instead it is enough if we add the `--include-build` to the CI-definitions (.travis.yml and java-windows.yml). So basically duplicate the existing build-steps/jobs and add --include-build. for instance, for mpp in java-windows.yml:
```
            -   name: Build sample mpp with current Atrium
                run: samples\multiplatform\gradlew -p samples\multiplatform build --include-build ../../../
```

Reply:
Wait wait... maybe gradle is wrong (let's hope). Maybe this approach works but you surely need to include Atrium as well (and make atrium-samples-test an independent project entirely, i.e. do not add it to settings.gradle in the root of the project). In order to test if it works, I advice you to change apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/anyAssertions.kt use the following for toBe
```
fun <T> Expect<T>.toBe(expected: T) = addAssertion(ExpectImpl.any.notToBe(this, expected))
```
if the composite build works, then the tests should be executed and fail.

Reply:
what is the meaning of ../../../?
where are we pointing to?
the include build was required in case we wanted a composite build, if we wont use it than why use --include-build?

Reply:
> Wait wait... maybe gradle is wrong (let's hope). Maybe this approach works but you surely need to include Atrium as well (and make atrium-samples-test an independent project entirely, i.e. do not add it to settings.gradle in the root of the project). In order to test if it works, I advice you to change apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/anyAssertions.kt use the following for toBe
> 
> ```
> fun <T> Expect<T>.toBe(expected: T) = addAssertion(ExpectImpl.any.notToBe(this, expected))
> ```
> 
> if the composite build works, then the tests should be executed and fail.

 I missed this comment...
Should we revert?

Reply:
> the include build was required in case we wanted a composite build, if we wont use it than why use --include-build?

Sorry for the confusion, that comment was before I saw the restriction of composite builds.

But to answer your other question

> what is the meaning of ../../../?

note the parameter -p in 
```
samples\multiplatform\gradlew -p samples\multiplatform build --include-build ../../../
```
so `../../../` is pointing back to the root of the project.

Reply:
arent we running from the root already? 
samples\... looks like reference from the root

Reply:
`-p` specifies the working directory for gradle and --include-build is relative to this working directory.

Reply:
understood
so leave the yml files without change or should I try and add the include-build param?

Reply:
I'll see if I can build up on your current work, tweaking gradle a bit so that we still can use composite build. But this would require `atrium-samples-test`. So for the moment you can wait until I get back to you.

Reply:
What you can do already in the meantime is:
- squash your commits, rebase on master and force push

Reply:
Nice, I have a working workaround, so you can proceed:
- add the following to settings.gradle:
   ```
    includeBuild('../../../') {
        dependencySubstitution {
            gradle.includedBuild('atrium').getAvailableModules().collect { pair -> pair.right.projectName }
                .findAll { it.endsWith("-jvm") }
                .forEach { projectName ->
                    def withoutSuffix = projectName.substring(0, projectName.length() - 4)
                    substitute module("ch.tutteli.atrium:$withoutSuffix") with project(":$projectName")
                }
        }
    }
    ```
- in build.gradle, change the dependsOn and always depend on `:build` (instead of test etc.)

Reply:
I can also merge your changes into a separate branch and push my changes, if you prefer (and then you can take up the work from there)